cli/tests/install.bats

```diff
@@ -534,3 +534,41 @@ EOF
   assert_success
   assert_output "$vim_store_path/bin/vim"
 }
+
+# ---------------------------------------------------------------------------- #
+
+@test "'flox install' info message when overriding included package" {
+  # This will be an included environment
+  "$FLOX_BIN" init -d included
+
+  # Install a package that will be overridden by the composer later
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
+    "$FLOX_BIN" install -d included hello
+
+  # Create the composing environment
+  "$FLOX_BIN" init -n "composer"
+
+  # Add the include to the manifest
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
+  "$FLOX_BIN" edit -f - <<- EOF
+version = 1
+
+[include]
+environments = [
+  { dir = "included" }
+]
+
+[options]
+systems = ["aarch64-darwin", "x86_64-darwin", "aarch64-linux", "x86_64-linux"]
+EOF
+
+  # Install the overriding package to the composer
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/hello.json" \
+    run "$FLOX_BIN" install hello
+
+  assert_output --partial "This environment now overrides package with id 'hello'"
```

Comment:
nit: I think(?) we've said we prefer not using `--partial` when possible, do we need it here?

Reply:
In general I agree. I needed it here because of the image I posted on Slack. I couldn't figure out what the diff was, and didn't feel like wasting my time trying to figure out if there's a weird Unicode error. I think it's fine.